### ISSUE TITLE
t/02-approver.t (well, yes, again...)

### ISSUE THREAD

**Issue by eserte** [rsimoes/WWW-eNom#11]
Last pass report on CPAN Testers was on 2016-03-14. Now the test suite fails:
```
#   Failed test 'Found resellerdocs.com domain admin'
#   at t/02-approver.t line 20.
#          got: 'ttvbfpxw@whoisprivacyprotect.com'
#     expected: 'pqqpyxkylv@whoisprivacyprotect.com'
# Looks like you failed 1 test of 1.
t/02-approver.t .............. 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/1 subtests 
```

**Comment by rsimoes** [comment]
Wow, okay. I'm just going to remove that test. Thanks for the report.

**Comment by eserte** [comment]
Maybe it would help to weaken the test, e.g. check only that the result looks vaguely like an email address. Something `like $admin, qr{.+\@.+}` ...

**Comment by rsimoes** [comment]
At this point the API command is returning an error of `The specified command is disabled - CERTGETAPPROVEREMAIL`. I'm not seeing any way to salvage testing this particular API call.

**Comment by rsimoes** [comment]
Fixed in v1.3.3
